dcp/api/job.py

```diff
@@ -1,13 +1,128 @@
 import pythonmonkey as pm
+import cloudpickle
+import dill
+import asyncio
+from ..js import utils
+from .. import dry
+from .job_serializers import Serializers
+from .job_env import Env
+from .job_modules import Modules
+from .job_fs import JobFS
+from collections.abc import Iterator
+from types import FunctionType
+import urllib.parse
+from .pyodide_work_function import work_function_string
+
 
 def job_maker(super_class):
     class Job(super_class):
-        #def exec(self, *args, **kwargs):
-        #    # TODO change behaviour to match spec
-        #    #print("overidden by Job hook")
```

Comment:
NOTE in a comment that this differs from dcp-client.js ' Job constructor args, please explicitly mention this Python Job construyctor takes the same args as compute.for...

Reply:
you can mention the constructor is currently undefined behaviour in dcp-clientjs anyways

Reply:
or maybe it doesn't make sense to mention that idk, do whatever you think is best

Reply:
This has changed now that the Job API class takes a js_ref in its contructor instead of args to compute_for